Python/marshal.c

```diff
@@ -1345,6 +1347,9 @@ r_object(RFILE *p)
 
             /* XXX ignore long->int overflows for now */
             argcount = (int)r_long(p);
+            if (PyErr_Occurred())
```

Comment:
Another  place that nees curly braces.

I'm going to stop commenting on these to cut down on spam, if you want me to point them out I can do another pass where I note all of them.

Reply:
In this case I have matched the style of the files to not make the diff huge. The PR is already gigantic so I prefer to fix it later (if fixed at all).

Reply:
You don't need to change all the braces in the file, but IIRC pure-refactoring PRs are not generally accepted for CPython, so you're supposed to "clean up as you go".

I don't see any particular reason to try to maintain stylistic consistency around braces use even within a function, so I don't think it would make the diff much bigger if you just follow PEP 7 within the code you are adding or changing.

Reply:
There is a big chain of conditionals in this function (and the file is fairly inconsistent). I still prefer not to do that refactoring in this PR.

Reply:
I was just suggesting that in this part you added, you do:

```C
if (PyErr_Occurred()) {
    goto code_error;
}
posonlyargcount = (int)r_long(p);
```

You don't need to refactor all the other ones in the chain. If anything I think consistency hurts here because it discourages people who touch this file from adding the braces.

Reply:
Ok, done in a85a43f